PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -822,10 +822,8 @@ private extension HTTPClientTests {
                                                                                  options: []) as? [String: Any])
             return try XCTUnwrap(requestBodyDict["requestNumber"] as? Int)
         } catch {
-            XCTFail(error.localizedDescription)
```

Comment:
I meant to include the `XCTFail` though, so the result is the test failing instead of crashing.